packages/npm/@amazeelabs/scaffold/test.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Clean up any leftover test directory from previous failed runs.
+rm -rf test
+
+# Create the test directory and change there.
+mkdir test
+cd test || exit 1
+
+# Initiate a git repository.
+git init
+echo "node_modules" >> .gitignore
+
+# Initiate a new node package.
+yarn init -y
+
+# Install the scaffold package.
+yarn add ../
+
+# Run the initial scaffold to install dotfiles and scripts.
+yarn amazee-scaffold
+yarn husky install && rm -rf .husky
```

Comment:
So husky should isn't managed by amazee-scaffold?

Reply:
I did not fully understand why we have to run `yarn husky install` again 😬 
Do you have an idea how we could manage it by `amazee-scaffold`?

Reply:
Make `yarn amazee-scaffold` run `yarn husky install && rm -rf .husky` for us?